agent/dns.go

```diff
@@ -673,6 +682,40 @@ RPC:
 	}
 }
 
+// addNSAndARecordsForDomain uses the agent's advertise address to
+func (d *DNSServer) addNSAndARecordsForDomain(msg *dns.Msg) {
```

Comment:
`s/addNSAndARecordsForDomain/addAuthority/g`

Also the comment isn't done yet.